### ISSUE TITLE
[Feature Request] Low resolution image preview

### ISSUE THREAD

**Issue by carlosdariodev** [FilenCloudDienste/filen-drive#49]
It would be nice to see a low res preview of an image when it's loading, with some kind of visual feedback that shows the progress of load of the real image.

**Comment by DanielProg39** [comment]
This would mean slower upload time and more storage used. With hundreds of photos it may become a real problem

**Comment by carlosdariodev** [comment]
But it is already implemented right? The JPGs have a thumbnail image already, or do you use the full image as a thumbnail?

In case it's the latter, thumbnails improve user experience, since it will increase the performance, plus, a thumbnail weights almost nothing, and you can also encrypt them and save them in the users computer. Also, I assume mega.io is the main competitor of filen.io, and they do it

**Comment by DanielProg39** [comment]
Yes, it actually makes sense since thumbnails are already implemented. You're right, the thumbnails are created as resized images of about 500x500px. I got it wrong the first time because I was thinking of creating new ones at a higher scale. Nevertheless, it makes sense to show existing thumbnails for a full image preview, and their size is pretty good for any screen size.

**Comment by Dwynr** [comment]
Agreed, could just use the locally saved thumbnail as a "loading placeholder" while the real image loads. We do it like that in the mobile app. Will add it to my todo list.

**Comment by carlosdariodev** [comment]
If you do implement it, please make sure to add some kind of user feedback that shows that the real image is loading, maybe a loading bar would do it, since you probably know the real size of the image (not in the middle of the screen, top or bottom would be ideal)
